package.json

```diff
@@ -16,9 +16,9 @@
     "url": "git://github.com/jcreigno/nodejs-mail-notifier.git"
   },
   "dependencies": {
-    "async": "^2.5.0",
-    "debug": "^2.2.0",
-    "imap": "~0.8.9",
+    "async": "^2.6.2",
+    "debug": "^4.1.1",
+    "imap": "~0.8.19",
     "mailparser": "0.6.2"
```

Comment:
2.7.1 is latest now

Reply:
I think that many of packes there was updated from time when I created this pull request. 

You are invited add new pull request with updates.